src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -73,4 +87,62 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
                 return false;
         }
     }
+
+    public async promptForCredentials(): Promise<{ username: string, password: string }> {
```

Comment:
Since you're already moving this code around, I've got some suggestions
1. Move the code in this method (promptForCredentials) to `getPostgresCredentials.ts`. That file has very little code and this file does not. The only call into the tree item would be for `persistServer`, which you could make public
1. Rename `persistServer` to `setCredentials`, to match `getCredentialsFromKeytar ` (I'm indifferent on whether or not it has 'keytar' in the name)
1. Rename `getPostgresCredentials.ts` to `enterPostgresCredentials.ts` to match the title of the command in package.json
1. Move `enterPostgresCredentials.ts` into a commands folder (Related to https://github.com/microsoft/vscode-cosmosdb/pull/1394#discussion_r400557852)